packages/web/components/Dashboard/Nav/Nav.tsx

```diff
@@ -130,7 +130,10 @@ const Nav: React.FC<Props> = ({ expanded, collapse, disableLargeNav }) => {
           {currentUser && (
             <>
               <div className="nav-top">
-                <Link href={`/dashboard/profile/[id]`} as={`/dashboard/profile/${currentUser.id}`}>
+                <Link
+                  href={`/dashboard/user/[handle]`}
+                  as={`/dashboard/user/${currentUser.handle}`}
```

Comment:
I see we had the ‘as’ here before but we should be able to get rid of it, just use the href field. 

Also I’d personally like to get “dashboard” out of that URL but that’s just me.